cmd/tendermint/commands/root.go

```diff
@@ -29,12 +29,20 @@ func registerFlagsRootCmd(cmd *cobra.Command) {
 
 // ParseConfig retrieves the default environment configuration,
 // sets up the Tendermint root and ensures that the root exists
-func ParseConfig() (*cfg.Config, error) {
+func ParseConfig(cmd *cobra.Command) (*cfg.Config, error) {
```

Comment:
I think it would be better to pass just the home path string instead of the entire command.